test/integration/analyzers/nodejs_test.go

```diff
@@ -0,0 +1,78 @@
+package nodejsanalyzer
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/fossas/fossa-cli/test/project_initializer/node"
+	"github.com/fossas/fossa-cli/test/testtools"
+	"github.com/stretchr/testify/assert"
+)
+
+var nodeInitializer = node.New()
+
+func TestMain(m *testing.M) {
```

Comment:
`TestMain` runs once and controls running of the test suite. after calling `flag.Parse`, we can short circuit all tests here if not running integration tests